website/docs/r/monitor.html.markdown

```diff
@@ -49,13 +49,13 @@ resource "datadog_monitor" "foo" {
 
 The following arguments are supported:
 
-* `type` - (Required) The type of the monitor, chosen from:
+* `type` - (Required) The type of the monitor. The mapping from these types to the in app Monitor types can be found in our [documentation](https://docs.datadoghq.com/api/?lang=python#create-a-monitor) Available options to choose from are:
```

Comment:
Two notes here:

* Not a native speaker, but "to the in app Monitor types" doesn't make much sense to me. Did you mean to use "to the types in Monitor app"?
* "our documentation" is ambiguous here IMO. Let's use "Datadog API documentation"